hnn_core/gui/gui.py

```diff
@@ -519,56 +308,52 @@ def _init_ui_components(self):
         self._simulation_status_bar = HTML(
             value=self._simulation_status_contents['not_running'])
 
-        # footer
-        self._footer = VBox([
-            HBox([
-                HBox([
-                    self.run_button, self.load_button, self.clear_button,
-                    self.delete_drive_button
-                ]),
-                self.widget_viz_layout_selection,
-            ]),
-            HBox([self._log_out], layout=self.layout['log_out']),
-            self._simulation_status_bar
-        ])
+        self._log_window = HBox([self._log_out], layout=self.layout['log_out'])
+        self._operation_buttons = HBox([self.run_button,
+                                        self.load_button,
+                                        self.clear_button,
+                                        self.delete_drive_button],
+                                       layout=self.layout['operation_box'])
         # title
         self._header = HTML(value=f"""<div
             style='background:{self.layout['theme_color']};
             text-align:center;color:white;'>
             HUMAN NEOCORTICAL NEUROSOLVER</div>""")
 
-        # visualiation components
-        self._viz_grid = {
-            "L-R": None,
-            "U-D": None,
-        }
-
     @property
     def analysis_config(self):
         """Provides everything viz window needs except for the data."""
         return {
-            "style": self.layout['visualization_window'],
-            "layout": self.widget_viz_layout_selection.value,
-            "max_spectral_frequency": self.widget_max_spectral_frequency.value,
-            "dipole_scaling": self.widget_dipole_scaling.value,
-            "dipole_smooth": self.widget_dipole_smooth.value,
-            "spectrogram_cm": self.widget_spectrogram_cm.value,
+            "viz_style": self.layout['visualization_output'],
             # widgets
-            "viz_grid": self._viz_grid,
             "viz_window": self._visualization_window,
             "plot_outputs": self.plot_outputs_dict,
             "plot_dropdowns": self.plot_dropdown_types_dict,
             "plot_sim_selections": self.plot_sim_selections_dict,
             "current_sim_name": self.widget_simulation_name.value,
         }
 
-    def load_parameters(self, params_fname=None):
+    @property
+    def data(self):
+        """Provides all GUI relate data."""
+        return {
+            "simulation_data": self.simulation_data,
+            "figure_trees": self.figure_trees,  # all saved fig conf
+            "vis_plot_tree": self.vis_plot_tree,  # unsaved fig conf
+            "fig_objs": self.fig_objs,
+            # derived data property
+            "fig_keys":
+            set([ftree[1]['fig_key'] for ftree in self.figure_trees.items()] +
```

Comment:
what's going on here??! is this still relevant?

Reply:
Apologies I forgot to delete this part, they're irrelevant now.